aws/resource_aws_spot_fleet_request.go

```diff
@@ -311,7 +313,94 @@ func resourceAwsSpotFleetRequest() *schema.Resource {
 				},
 				Set: hashLaunchSpecification,
 			},
-			// Everything on a spot fleet is ForceNew except target_capacity
+			"launch_template_configs": {
```

Comment:
Nit: For configuration block attributes, we prefer the singular noun so configurations look like the following:

```hcl
resource "aws_spot_fleet_request" "example" {
  launch_template_config {
    # ...
  }

  launch_template_config {
    # ...
  }
}
```